test/resources/TelephonyCredentials.spec.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var utils = require('../../testUtils');
+var telnyx = utils.getTelnyxMock();
+var expect = require('chai').expect;
+
+var TEST_AUTH_KEY = utils.getUserTelnyxKey();
+
+var credentialCreateData = {
+  credential: {
+    connection_id: '1474011037387720344',
+  },
+};
+
+describe('TelephonyCredentials Resource', function () {
+  describe('create', function () {
+    function responseFn(response) {
+      expect(response.data).to.have.property('created_at');
+      expect(response.data).to.have.property('expired');
+      expect(response.data).to.have.property('expires_at');
+      expect(response.data).to.have.property('id');
+      expect(response.data).to.have.property('name');
+      expect(response.data).to.have.property('record_type');
+      expect(response.data).to.have.property('resource_id');
+      expect(response.data).to.have.property('sip_password');
+      expect(response.data).to.have.property('sip_username');
+    }
+
+    it('Sends the correct request', function () {
+      return telnyx.telephonyCredentials.create(credentialCreateData).then(responseFn);
```

Comment:
I don't see this being mocked in the PR, is the test running against the actual API?

Reply:
This project has an API mock that you need to run separately and concurrently:

You can check here https://github.com/team-telnyx/telnyx-node#development